projects/angular-cld/karma.conf.js

```diff
@@ -25,7 +26,7 @@ module.exports = function (config) {
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: true,
-    browsers: ['Chrome'],
+    browsers: ['Chrome', 'Firefox'],
```

Comment:
@strausr , is that related to this fix?

Reply:
Yes, we're testing the output in Firefox... The tests require it

Reply:
Specifically for this feature?